### ISSUE TITLE
Switch to the new libkv

### ISSUE THREAD

**Issue by 0xbaadf00d** [containous/traefik#1975]
Alexandre Beslic (@abronan) one maintainer of libkv just set the project alive by merging lot of PR (etcd fixes, added Redis, ...) and forking the project to continue to upgrade this project.

It would be interresting for Traefic to switch to this new version.

URL : https://github.com/abronan/libkv

**Comment by ldez** [comment]
https://github.com/containous/traefik/issues/926#issuecomment-323348372

**Comment by dtomcej** [comment]
wow, there has been a LOT of activity in that repo in the last 2 weeks.

I look forward to awesome etcd3 support!

**Comment by trusch** [comment]
etcd v3 is ready in this repo. bump :)

**Comment by abronan** [comment]
@trusch afaik they are working on it but trying to fix a few remaining issues (as I haven't had the time to test etcd v3 support with traefik yet). This could unblock soon.

**Comment by zyclonite** [comment]
looks like alexandre is in the process of (or even already done) renaming the libkv to valkeyrie (https://github.com/abronan/valkeyrie)

... and continuing to maintain only this repository

**Comment by zicklag** [comment]
So does this mean that Redis is a supported KV backend now? It isn't in the [documentation](https://docs.traefik.io/user-guide/kv-config/).
